### PR TITLE
RDBC-522 Support for https

### PR DESCRIPTION
**Pull request by poissoncorp**

### CODE REVIEW

File: .github/workflows/add_ca.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import certifi
+
+catfile = certifi.where()
+print(catfile)
```
Comment (ml054) [inline .github/workflows/add_ca.py]: add info what you print here

File: .github/workflows/add_ca.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import certifi
+
+catfile = certifi.where()
```
Comment (ml054) [inline .github/workflows/add_ca.py]: it is cafile, not catfile

🐱

Comment (poissoncorp) [reply .github/workflows/add_ca.py]: that's true ¯\_(ツ)_/¯

File: pyravendb/tests/driver/raven_test_driver.py [hunk 3]
```diff
@@ -46,13 +46,12 @@ def _run_server_internal(
...
         if url is None:
             self._report_info("Url is None")
-
             try:
                 process.kill()
             except Exception as e:
                 self._report_error(e)
 
-            raise RuntimeError("Unable to start server")
+            raise RuntimeError(f"Unable to start server GH {read_lines}")  # todo : remove
```
Comment (ml054) [inline pyravendb/tests/driver/raven_test_driver.py]: remove?

Comment (poissoncorp) [reply pyravendb/tests/driver/raven_test_driver.py]: GH Actions debug, 'll remove before merge

**Pull request merged**